io/net/virt-net/network_virtualization.py

```diff
@@ -35,6 +35,9 @@
 from avocado.utils.network.hosts import LocalHost
 from avocado.utils.ssh import Session
 from avocado.utils import wait
+import pexpect
```

Comment:
remove the above line  pexpect is not explicitly used in the code .. this will resolve the travis fail